src/main/java/org/radarcns/phone/PhoneSensorManager.java

```diff
@@ -104,6 +137,30 @@ public void start(@NonNull final Set<String> acceptableIds) {
             logger.warn("Phone Light sensor not found");
         }
 
+        // Gyroscope
+        if (sensorManager.getDefaultSensor(Sensor.TYPE_GYROSCOPE) != null) {
+            Sensor gyroscope = sensorManager.getDefaultSensor(Sensor.TYPE_GYROSCOPE);
+            sensorManager.registerListener(this, gyroscope, SENSOR_DELAY_DEFAULT);
+        } else {
+            logger.warn("Phone Gyroscope not found");
+        }
+
```

Comment:
This code block is repeated quite a few times, perhaps a `private static void registerSensor(SensorManager sensorManager, int sensorType, String sensorName)` could be used instead?